### PR TITLE
Rename and Unexport isRuntimeMessage Task 10256

### PR DESCRIPTION
**Pull request by MarioJGMsoft**
## Description

Rename and unexport `isRuntimeMessage` function from containerRuntime.ts
isRuntimeMessage was renamed to isUnpackedRuntimeMessage

### CODE REVIEW

File: packages/runtime/container-runtime/src/index.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ export {
 	IContainerRuntimeOptions,
 	loadContainerRuntime,
 	LoadContainerRuntimeParams,
-	isRuntimeMessage,
```
Comment (tyler-cai-microsoft) [inline packages/runtime/container-runtime/src/index.ts:15]: This part of the change broke the type tests.

**Comment by MarioJGMsoft** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: packages/runtime/container-runtime/src/test/types/validateContainerRuntimePrevious.generated.ts [hunk 2]
```diff
@@ -382,6 +382,7 @@ declare type current_as_old_for_Function_detectOutboundReferences = requireAssig
  * typeValidation.broken:
  * "Function_isRuntimeMessage": {"backCompat": false}
  */
+// @ts-expect-error compatibility expected to be broken
```
Comment (markfields) [inline packages/runtime/container-runtime/src/test/types/validateContainerRuntimePrevious.generated.ts:385]: @CraigMacomber - I expected this to switch to the "Removed" modality, not merely saying it is incompatible.  Did something change?  I walked Mario through making the change and building locally, not sure if there's some secret incantation we missed, or what. Do you know?

**Review by markfields — APPROVED** [review]
LGTM!

**Pull request merged**